src/mappings/utils/cumulativeVolumes.ts

```diff
@@ -192,6 +193,7 @@ export async function updateCumulativeVolumesForCurrencyPair(
                 type,
                 tillTimestamp: timestamp,
                 amount: existingCumulativeVolumeForCollateral + amount,
+                amountHuman: await convertAmountToHuman(collateralCurrency, amount),
```

Comment:
I think this should be
```
amountHuman: await convertAmountToHuman(collateralCurrency, existingCumulativeVolumeForCollateral + amount),
```

ie. see line above for atomic amount

Reply:
Fixed